openreview/conference/templates/programchairWebfield.js

```diff
@@ -157,6 +157,14 @@ var main = function() {
       }
     }
 
+    conferenceStatusData.reviewerOptions = _.map(conferenceStatusData.reviewers, function(r) {
+      var profile = findProfile(conferenceStatusData.profiles, r);
+      return {
+        id: r,
+        description: view.prettyId(profile.name) + ' - (' + profile.allEmails + ')'
```

Comment:
Won't this list all the emails a user has on their profile? I think we should just show `profile.email` which will be equal to the preferred email, or the first email in the email list.

I also don't think the dash in between is necessary - the () separate it enough and it's on a new line anyway

Reply:
I prefer to show all the emails(confirmed) so users can search reviewers by any email. The issue that Mohit this morning the reviewer was assigned using an alternate email instead of preferred.

Reply:
OK, well in that case I think it would be better to do `profile.allEmails.join(', ')` so that the text will wrap.